src/lib/storage/storage_manager.cpp

```diff
@@ -7,45 +7,57 @@
 
 #include "utils/assert.hpp"
 
+#include <boost/range/combine.hpp>
+
 namespace opossum {
 
 StorageManager& StorageManager::get() {
-  return *(new StorageManager());
-  // A really hacky fix to get the tests to run - replace this with your implementation
+  static StorageManager instance;
```

Comment:
Here, you could use *auto to stick*.

Reply:
Do you mean something like this? Also, could this solution override the instance with a new `StorageManager` for each invocation of `get` (which would be problematic for us)?

```suggestion
  static auto instance = StorageManager();
```

Reply:
This works on our end. We believe this does not alter the ``instance`` due to its static nature.

However, we propose doing the following to keep in line with the coding style guidelines:
`static auto instance = StorageManager{};`

Reply:
Thank you for clarifying. Will implement :)